Updating PingPong example

Andres-Pardo-R opened the pull request:
Tested the usability of the pingpong (python, nodejs, C and C#) examples and updating the files needed (Installation.md and README.md)

Andres-Pardo-R committed:
Updating application example

Testing the usability of the application examples pingpong (python, nodejs, C and C#) and updating the files needed (Installation.md and README.md)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/industrial-edge/pingpong-csharp?pullRequest=18) <br/>All committers have signed the CLA.

HummelN7872 commented on README.md:
```diff
@@ -98,6 +97,7 @@ The finished flow is available [here](/src/Flow_Pingpong_Test.json) and can be i
   - [Industrial Edge landing page](https://new.siemens.com/global/en/products/automation/topic-areas/industrial-edge/simatic-edge.html)
   - [Industrial Edge GitHub page](https://github.com/industrial-edge)
   - [Industrial Edge documentation page](https://docs.eu1.edge.siemens.cloud/index.html)
```
@Andres-Pardo-R the link to this documentation website is not longer needed as you added the new operations x website. Please remove this point

Andres-Pardo-R committed:
Update README.md

Deleted old link to industrial edge guide and corrected the version of the databus configurator
